src/xpcc/architecture/driver/atomic/queue.hpp

```diff
@@ -67,6 +67,9 @@ namespace xpcc
 			ALWAYS_INLINE bool
 			isFull() const;
 			
+			ALWAYS_INLINE bool
+			isNotFull() const { return not isFull(); }
```

Comment:
> I didn't mean to offend you in any way, I was just genuinely curious about the use of these operators.

Oh, don't worry, I didn't read it that way.

I was thinking of some past discussions with other developers. Very smart people, but somewhat… limited in their imagination. You know the "we've always done it this way"™ kind of mentality.

I like to 🔥 down the established conventions and build better ones in their place, but that's not always an option.